install

```diff
@@ -137,26 +142,32 @@ do_install()
   pushd src
     # Avoid downloading opencv3; we already installed it from homebrew.
     wstool merge file://$(pwd)/../${WS}.rosinstall
-    wstool remove opencv3
+    #wstool remove opencv3
```

Comment:
Please remove this stuff altogether vs just commenting it out— as of Melodic, opencv is back to using the system-packaged version with not entry in the rosdistro.

Reply:
From time to time we might want to be able to update the repos in a systematic way, especially after new releases. Should I add an input that controls whether we want to update it? The default value would be False.